interp/src/macros.rs

```diff
@@ -1,13 +1,14 @@
 /// Define a primitive
 ///
-/// ```
-/// primitive!(StdAdd[width](left: width, right: width) {
+/// ```ignore
```

Comment:
What's the `ignore` for?

Reply:
the doc test for the macros don't actually work and I didn't want to waste a bunch of time fixing them, probably should go back and do that now that the rest are actually working

Reply:
Okay I fixed the doc tests, turns out there were a bunch of places that had `crate` instead of `$crate`.